File: src/main/java/org/zenbot/szolnok/timetable/configuration/MongoConfiguration.java
package org.zenbot.szolnok.timetable.configuration;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@Configuration
@EnableMongoRepositories("org.zenbot.szolnok.timetable.dao")
public class MongoConfiguration {
}

File: szolnok-timetable-batch-news/src/main/kotlin/org/zenbot/szolnok/timetable/batch/news/configuration/MongoConfiguration.kt
package org.zenbot.szolnok.timetable.batch.news.configuration

import org.springframework.context.annotation.Configuration
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories

@Configuration
@EnableMongoRepositories("org.zenbot.szolnok.timetable.batch.news.dao")
class MongoConfiguration
